File: Suhyun/Level1/1월 2주차/자릿수 더하기/Add_digit.js
let n=123;
let str=String(n);
let answer=0;
let arr=str.split('');

arr.map(i => {
    answer=answer+parseInt(i)});

console.log(answer);


//프로그래머스용
function solution(n)
{
    let str=String(n);
    let answer=0;
    let arr=str.split('');

    arr.map(i => {
        answer=answer+parseInt(i)});

    return answer;

}

File: Suhyun/Level1/1월 2주차/자릿수 더하기/Add_digit.java
/**
 * 1. 자릿수만큼 반복문 돌리기
 * 2. %10으로 나눠서 각 자릿수 더하기
 */
public class Add_digit {
    public static void main(String[] args) {
        int n=1032;
        int answer=0;
        while(true){
            answer=answer+(n%10);
            n=n/10;
            if(n==0)break;
            
        }

        System.out.println(answer);

        int i = 10;
        int num = i * i;
        
    }
    
}